core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/clustering/hazelcast/HazelcastDistributedMapProvider.java

```diff
@@ -258,7 +262,7 @@ public void clear() {
             return new LinkedHashSet<Entry<K, V>>();
         }
 
-        public String getListenerId() {
+        public UUID getListenerId() {
```

Comment:
We shouldn't ideally change return types of public methods. Why was this required?